doc/Makefile

```diff
@@ -13,6 +13,11 @@ DT_TURBO_MODE ?= 0
 
 .PHONY: configure clean html html-fast latex pdf doxygen
 
+# For a build fast enough for interactive use, turn off all slow sphinx
```

Comment:
Why not add an "html-minimal" target that does just that?

Reply:
Done, thanks!